governance-exercises/VoteWizard_poll_exercise_02/README.md

```diff
@@ -0,0 +1,53 @@
+# VoteWizard Polling Exericse #2
+
+## Assignment #1: Determine what course of action you would take as a Governance Facilitator and communicate it clearly
+
+### The course of action I would take as a Governance Facilitator:
+
+1. I'd reply to Clive's Reply #1, explaining to him and the community the criteria I used to determine that the materials shared by Diane constitute reasonable suspicion (or significant suspicion). I would emphasize that the voters will ultimately make the decision.
```

Comment:
Arguably, voters are deciding whether it is sufficient suspicion or not, rather than whether to derecognize them directly.